packages/search-ui-app-search-connector/src/__tests__/AppSearchAPIConnector.test.ts

```diff
@@ -386,7 +386,17 @@ describe("AppSearchAPIConnector", () => {
             {
               all: [
                 {
-                  date_made: "yesterday"
+                  title: "Acadia"
```

Comment:
This change seems a little odd but *i believe* the correct behaviour here. The test case here has two filters in the request state and one filter in the queryConfig. Previously it was only providing the filter in the `queryConfig` to the API. Now its passing all three filters to the API. I believe it should be passing all three filters to the API.

Reply:
Thanks for the spot! So the testcase is incorrect here, the `date_made: "yesterday"` should of been present in `requestState.filters` too, as searchDriver runs `mergeFilter` which mutates the `requestState.filters` with filters from the `queryConfig.filters`